src/components/Navbar.js

```diff
@@ -0,0 +1,11 @@
+function Navbar(){
+    return(
+        
+    <nav class="navbar navbar-dark bg-primary mb-3">
```

Comment:
you can put NavLink inside a div tag to see every country in  a single rrow :

`<div>
            <Link to= {country.alpha3Code}>{country.name.common}</Link>
            </div>`